modules/openapi-generator/src/main/resources/swift5/modelEnum.mustache

```diff
@@ -1,7 +1,32 @@
-{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{classname}}: {{dataType}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable {
+{{#nonPublicApi}}internal{{/nonPublicApi}}{{^nonPublicApi}}public{{/nonPublicApi}} enum {{classname}}: {{dataType}}, {{#useVapor}}Content, Hashable{{/useVapor}}{{^useVapor}}Codable{{/useVapor}}, CaseIterable{{^generateFrozenEnums}}{{#isInteger}}, CaseIterableDefaultsLast{{/isInteger}}{{#isFloat}}, CaseIterableDefaultsLast{{/isFloat}}{{#isDouble}}, CaseIterableDefaultsLast{{/isDouble}}{{#isString}}, CaseIterableDefaultsLast{{/isString}}{{/generateFrozenEnums}} {
 {{#allowableValues}}
 {{#enumVars}}
     case {{{name}}} = {{{value}}}
 {{/enumVars}}
 {{/allowableValues}}
+{{^generateFrozenEnums}}
+    // This case is a catch-all generated by OpenAPI such that the enum is "non-frozen".
+    // If new enum cases are added that are unknown to the spec/client, they are safely
+    // decoded to this case. The raw value of this case is a dummy value that attempts
+    // to avoids collisions with previously specified cases.
+{{#isString}}
+    case unknownDefaultOpenApi = "unknown_default_open_api"
+{{/isString}}
+{{#isNumeric}}
+    //
+    // 192, used to calculate the raw value, was the Swift Evolution proposal for
+    // frozen/non-frozen enums.
+    // [SE-0192](https://github.com/apple/swift-evolution/blob/master/proposals/0192-non-exhaustive-enums.md)
+    //
+{{/isNumeric}}
+{{#isInteger}}
+    case unknownDefaultOpenApi = -11184809 // -(Int.min / 192)
+{{/isInteger}}
+{{#isDouble}}
+    case unknownDefaultOpenApi = -0.016362461737446838 // -(Double.pi / 192)
```

Comment:
@jarrodparkes maybe it's a good idea to use always the same number -11184809 in all the cases?